.github/workflows/docker.yaml

```diff
@@ -12,18 +12,23 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Login to registry
-        run: |
-          docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD $DOCKER_REGISTRY_URL
-        env:
-          DOCKER_REGISTRY_URL: docker.pkg.github.com
```

Comment:
GitHub Packages Docker Registry (aka docker.pkg.github.com) is deprecated, see https://github.com/marketplace/actions/docker-login#github-packages-docker-registry